nslsii/__init__.py

```diff
@@ -223,7 +224,15 @@ def configure_base(
         configure_ipython_logging(exception_logger=log_exception, ipython=ipython)
 
     if publish_documents_with_kafka:
-        configure_kafka_publisher(RE, beamline_name=broker_name)
+        if isinstance(publish_documents_with_kafka, str):
+            configure_kafka_publisher(RE, beamline_name=publish_documents_with_kafka)
+        elif hasattr(broker_name, "name"):
+            configure_kafka_publisher(RE, beamline_name=broker_name.name)
```

Comment:
In #208, @gwbischof proposed to `.lower()` the name just in case it's not lower-case for some reason.
```suggestion
            configure_kafka_publisher(RE, beamline_name=broker_name.name.lower())
```